PyTado/interface/api/my_tado.py

```diff
@@ -400,6 +400,20 @@ def change_presence(self, presence: Presence) -> None:
 
         self._http.request(request)
 
+    def set_child_lock(self, device_id: int, child_lock: bool) -> None:
+        """
+        Sets the child lock on a device
+        """
+
+        request = TadoRequest()
+        request.command = "childLock"
+        request.action = Action.CHANGE
+        request.device = device_id
+        request.domain = Domain.DEVICES
+        request.payload = {"childLockEnabled": child_lock}
+
+        self._http.request(request)
+
```

Comment:
Here is the same request for Tado X:

```
PATCH https://hops.tado.com/homes/1745644/roomsAndDevices/devices/VA3250358016?ngsw-bypass=true
{"childLockEnabled":true}
StatusCode: 204
```

You can add it to hops_tado.py if you like :D